.github/workflows/lint-test.yml

```diff
@@ -189,11 +184,10 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Install dependencies
-      run: pdm install -dG test  # needed for coverage
+      run: uv sync --no-default-groups --group test --no-editable
```

Comment:
Noticed while looking at #1327, this step shouldn't be necessary; coverage was removed a while back, and nox installs dependencies in a separate venv anyway.